### PR TITLE
File/folder housekeeping and AST update

### PR DESCRIPTION
**Pull request by ocornoc**

### CODE REVIEW

File: information/ast/cylon_ast.md [hunk 1]
```diff
@@ -137,3 +137,8 @@ Either a local variable or a data field in yolol. Data field identifiers are pre
 **Required keys:** `num: String`
 
 A number in yolol. Is encoded as a string due to the hyper-specific decimal rules binding compliant yolol numbers.
+
+### `value::string`
+**Required keys:** `str: String``
```
Comment (Jerald) [inline information/ast/cylon_ast.md]: Gotta remove the second backtick there. Isn't being shown in code-text

Comment (ocornoc) [reply information/ast/cylon_ast.md]: Alright, will resolve.

Comment (ocornoc) [reply information/ast/cylon_ast.md]: `type_metadata`?

Comment (Jerald) [reply information/ast/cylon_ast.md]: That sounds like a good name

**Pull request merged**